Tools/WinMLRunner/WinMLRunner.vcxproj

```diff
@@ -211,6 +213,7 @@
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <AdditionalDependencies>WindowsApp.lib; mincore.lib; DXGI.lib</AdditionalDependencies>
+      <DelayLoadDLLs>"ext-ms-win-dxcore-l1-1-0.dll"</DelayLoadDLLs>
```

Comment:
Hmm, have you asked Jeff W if this is the correct DLL? It's kinda weird for an app to directly link to specifically versioned ext-ms DLL's. (usually only OS components do that)

Reply:
@jeffwick-msft : Is this proper?

Reply:
Correct, this should be removed.

Reply:
If I remove this, then WinMLRunner won't be able to run on builds that don't have DxCore. 

I get this error:
![image](https://user-images.githubusercontent.com/16694211/56393568-b09f1280-61e9-11e9-8e1f-0369f58f6e7d.png)